src/graph/split-helpers.hh

```diff
@@ -49,7 +49,9 @@ hb_vector_t<unsigned> actuate_subtable_split (Context& split_context,
     if (id == (unsigned) -1)
     {
       new_objects.reset ();
-      new_objects.allocated = -1; // mark error
+      if (!new_objects.in_error()) {
+        new_objects.allocated = -1; // mark error
```

Comment:
Setting `allocated` is definitely not a good idea. We should add a method to vector for whatever this means. There's `set_error()`.